distributed/comm/tests/test_comms.py

```diff
@@ -1275,5 +1275,5 @@ def test_register_backend_entrypoint():
         "udp", mod.__name__, attrs=["UDPBackend"], dist=dist
     )
 
-    result = get_backend("udp")
+    result = get_backend("udp", require=False)
```

Comment:
Overall this looks good. Can we add a small comment here as to why we need `require=False` here?

Reply:
done